app/assets/stylesheets/announcements.scss

```diff
@@ -0,0 +1,40 @@
+.announcements {
```

Comment:
Won't be required after adding proper bootstrap classes

Reply:
made changes accordingly